### ISSUE TITLE
It seems there is no hardware requirements for proxy servers

### ISSUE THREAD

**Issue by vzhestkov** [SUSE/doc-susemanager#659]
There is a note on the page:
If you want to install SUSE Manager Proxy on a virtual machine, ensure your virtual machine has enough disk space and RAM by checking the requirements at Hardware.

But the link under Hardware word is showing only Server hardware requirements, not Proxy.

**Comment by Loquacity** [comment]
@mcalmer Can you comment on Proxy requirements?

**Comment by mcalmer** [comment]
Hard to say, but let me try to put some numbers out of the dark here:

- CPU: Minimum 2 dedicated 64-bit CPU cores
- RAM:
  * Test Server Minimum 2 GB
  * Production Server Minimum 8 GB
- Disk Space:
  * / (root) 24 GB - JeOS default
  * /var/cache (squid cache) minimum 100 GB (when more is available, squid can cache more)
  * @jsrain : maybe something additional for /srv in case of a retail branch server ?

@mbrookhuis @dvosburg : What are your experience with setting up a proxy? Are these numbers sufficient as minimal starting values?

**Comment by jsrain** [comment]
@mcalmer Yes, please, add a note that /src should contain enough disk space in order to host the synced images. I know that this statement is rather vague, however, the size of images can vary a lot, so does the number of them which is expected to be hosted.
If you really want a number, I would suggest 100GB could be reasonable.
